src/include/utils/rangerrest.h

```diff
@@ -80,6 +80,13 @@ typedef struct RangerPrivilegeResults
 {
   RangerACLResult result;
   Oid relOid;
+
+  /* 
+   * string_hash of access[i] field of ranger request 
+   * use the sigh to identify each resource result
+   */ 
+  uint32 resource_sign;
+  uint32 privilege_sign;
```

Comment:
On a batch ranger acl check request/response:
The acl check result order isn't the same as request order.
When reponse's resource&privilege field is same as request, then we consider them is a pair.

So we need storage resource&privilege sign in here.